plugins/services/src/js/structs/Framework.js

```diff
@@ -20,6 +20,14 @@ module.exports = class Framework extends Application {
     return this.get("slave_ids");
   }
 
+  getTasks() {
```

Comment:
@MatApple in that case we are only contributing for more legacy because the consistency here is only creating more of what we don't want instead moving forward as we go because it's very hard for us to change all together.

If we can, using the object.notation would still be consistent only not to structs and even easier to discover since the stack and path is direct.